src/django_rich/test.py

```diff
@@ -161,18 +185,97 @@ class RichTestRunner(DiscoverRunner.test_runner):  # type: ignore [misc]
     resultclass = RichTextTestResult
 
 
+class RichTestSuite(unittest.TestSuite):
+    def _handleModuleFixture(self, test, result):
+        timing_start = time.perf_counter_ns()
+        super()._handleModuleFixture(test, result)
+        total = (time.perf_counter_ns() - timing_start) / 1e9
+        if total >= 0.005:
```

Comment:
This section is now repeated three times. I think we could make this into a separate function?